src/crypto/local.js

```diff
@@ -34,7 +34,7 @@ let CURVE_H2C_METHOD;
 
 /**
  * Sets the curve parameters for the current session based on the contents of
- * ACTIVE_CONFIG.h2c-params
+ * activeConfig.h2c-params
```

Comment:
> Why this function encodes to hex? I think this changes the previous seed generation method.

I think this will be changed once the PR is rebased onto the latest commit (for handling compressed points).

Reply:
Fixed by rebase.